server/helpers/ffmpeg-utils.ts

```diff
@@ -138,6 +148,12 @@ function transcode (options: TranscodeOptions) {
       command = command.withFPS(fps)
     }
 
+    // Constrained Encoding (VBV)
+    // https://slhck.info/video/2017/03/01/rate-control.html
+    // https://trac.ffmpeg.org/wiki/Limiting%20the%20output%20bitrate
+    const targetBitrate = getTargetBitrate(options.resolution, fps, VIDEO_TRANSCODING_FPS)
+    command.outputOptions([`-b:v ${ targetBitrate }`, `-maxrate ${ targetBitrate }`, `-bufsize ${ targetBitrate * 2 }`])
```

Comment:
One small question: what happens if the input file has an average bitrate < targetBitrate?

Reply:
@Nutomic won't the `-b:v ${ targetBitrate }` increase the bitrate of the video if it is lower than the targetBitrate? I feel like we should have just kept `maxrate` and `bufsize` to benefit from ffmpeg's automatic tuning.

Reply:
Damn you're right. I just checked and it works fine if you remove the `-b:v` parameter (tested with low bitrate, high bitrate and very high bitrate like in the unit test). I was having some bug during development, where FFmpeg generated videos with a too high bitrate for no reason, but that is gone now.

Can one of you just make a commit to remove that parameter?

Reply:
Sure :) no worries, it's my fault for not double checking that